### ISSUE TITLE
Versions of extensions

### ISSUE THREAD

**Issue by christianp** [numbas/editor#664]
I've rewritten an extension, so questions written for the old version will no longer work.
I also regularly have to use my local server to test new changes to an extension, so I don't inadvertently break other people's questions while debugging.

The editor should store multiple versions of extensions, and references to extensions should include the version. Declaring a new version should be an explicit step. There should be the option to always use the "development" version, but this should not be the default.

**Comment by chrismgraham** [comment]
Good idea. For the extensions that sit on Github, should/could the versions be linked to github versions/branches?

**Comment by christianp** [comment]
that's related to #638
